cdc_rsync/server_arch.cc

```diff
@@ -106,53 +98,50 @@ std::string ServerArch::GetStartServerCommand(int exit_code_not_found,
       // a minor issue and means we display "Deploying server..." instead of
       // "Server not deployed. Deploying...";
       return RemoteUtil::QuoteForWindows(
-          absl::StrFormat("Set-StrictMode -Version 2; "
+          absl::StrFormat("powershell -Command \" "
+                          "Set-StrictMode -Version 2; "
                           "$ErrorActionPreference = 'Stop'; "
-                          "$server_dir = %s; "
-                          "$server_path = %s; "
-                          "$u=New-Item $server_dir -ItemType Directory -Force; "
-                          "if (-not (Test-Path -Path $server_path)) { "
+                          "if (-not (Test-Path -Path '%s')) { "
                           "  exit %i; "
                           "} "
-                          "& $server_path %s",
-                          server_dir, server_path, exit_code_not_found, args));
+                          "%s %s "
+                          "\"",
+                          server_path, exit_code_not_found, server_path, args));
     case Type::kLinux:
-      return absl::StrFormat(
-          "mkdir -p %s; if [ ! -f %s ]; then exit %i; fi; %s %s", server_dir,
-          server_path, exit_code_not_found, server_path, args);
+      return absl::StrFormat("if [ ! -f %s ]; then exit %i; fi; %s %s",
```

Comment:
You're right, but it would be a bit of work, and since those are internal APIs where we control the contents, I don't think it's worth it. The args are actually multiple arguments, so I'd have to split them into a vector first. This would mean also GameletComponent::ToCommandLineArgs would have to return a vector, which would again mean changes in a bunch of places, including cdc_stream.